R/dvhydrograph-render.R

```diff
@@ -24,34 +22,44 @@ createDvhydrographPlot <- function(data){
     endDate <- formatDates(data$reportMetadata$endDate) + hours(23) + minutes(45)
     plotDates <- seq(startDate, endDate, by=7*24*60*60)
     
-    plot_object <- gsplot(ylog=dvInfo$logAxis, yaxs='r') %>% 
-      grid(nx=0, ny=NULL, equilogs=FALSE, lty=3, col="gray") %>%
-      axis(1, at=plotDates, labels=format(plotDates, "%b\n%d"), padj=0.5) %>%
-      axis(2, reverse=isInverted) %>%
-      view(xlim=c(startDate, endDate)) %>% 
-      legend(location="below", cex=0.8, y.intersp=1.5) %>%
-      title(main="DV Hydrograph", 
-            ylab = paste0(data$firstDownChain$type, ", ", data$firstDownChain$units),
-            line=3)
+    plot_object <- gsplot(ylog = dvInfo$logAxis, yaxs = 'i') %>%
```

Comment:
These lines are showing highlighting as changed but do not show changes. Are you inserting windows newlines? Can you convert all back to unix and update? Helps with the review.